src/components/backend-ai-webui.ts

```diff
@@ -11,9 +11,6 @@ import {
   IronFlexFactors,
   IronPositioning,
 } from '../plastics/layout/iron-flex-layout-classes';
-import { Drawer } from '../plastics/mwc/mwc-drawer';
-import '../plastics/mwc/mwc-multi-select';
```

Comment:
@agatha197  "mwc-multi-select" is removed. Is "mwc-multi-select"  not used?

Reply:
It is imported but not used.